python/ray/util/sgd/v2/backends/backend.py

```diff
@@ -141,13 +153,54 @@ def get_next():
             if not all(r is None for r in results):
                 raise RuntimeError("Some workers returned results while "
                                    "others didn't. Make sure that "
-                                   "`sgd.report()` is called the same number "
-                                   "of times on all workers.")
+                                   "`sgd.report()` is called the same "
```

Comment:
Make sure `sgd.report()` and `sgd.save_checkpoint()` is called the same number of times on all workers.